libs/wingc/src/parser.rs

```diff
@@ -1394,37 +1391,34 @@ impl<'s> Parser<'s> {
 		let object_expr = self.get_child_field(nested_node, "object")?;
 
 		if let Some(property) = nested_node.child_by_field_name("property") {
-			let object_expr = if object_expr.kind() == "json_container_type" {
-				Expr::new(
+			if object_expr.kind() == "json_container_type" {
+				Ok(Expr::new(
 					ExprKind::Reference(Reference::TypeMember {
```

Comment:
todo: figure our why the double reference here didn't jsify into "Json.something().something()". Or did it?

Reply:
the reason is because of another bug in how `reference_to_udt` used to handle a `TypeReference` (it just ignored any property access on a reference containing this reference. Anyway not relevant anymore since we don't have `TypeReference`s anymore.